CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Deprecations:
 
 Enhancements:
  - Recognize read-only parameters by the `In` modreq (@zvirja, #406)
+ - NLogFactory will no longer load NLog.config by default, since NLog does it automatically (@snakefoot, #419)
```

Comment:
Perhaps this could be a tiny bit more specific:

```suggestion
 - NLogFactory's default constructor will no longer load NLog.config by default, since NLog does it automatically (@snakefoot, #419)
```

(And in case that the same change is applied to `ExtendedNLogFactory`&mdash;see my other review comment&mdash;that other class' default ctor should be mentioned here as well.)

Reply:
Good catch. Have now also updated `ExtendedNLogFactory` and added to ChangeLog.md